.env.template

```diff
@@ -1,6 +1,6 @@
-SOURCE_PROTOCOL=https
-SOURCE_DOMAIN=subdomain.example.com
-DESTINATION_PORT=8001
+SOURCE1_PROTOCOL=https
+SOURCE1_DOMAIN=subdomain.example.com
+DESTINATION1_PORT=8001
```

Comment:
I think SOURCE_PROTOCOL is fine here,

User can modify the `templates/default.conf.template` to support multiple domain using `/etc/hosts` where they can then use `server_name` and `/etc/hosts` to support multiple proxies.

Reply:
Maybe we can also push this changes in a different branch specifically for idmc-website-components

Reply:
I was thinking we could just create a fork on the idmc workspace